corpora/csip/metadata/mets-root/CSIP5/testCase.xml

```diff
@@ -1,23 +1,88 @@
-<!-- Root element for an individual test case, allows these to be wrapped into
-XML lists -->
-<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd"
-  testable="UNKNOWN">
-  <!-- Unique ID for the test case -->
-  <id specification="E-ARK CSIP" version="2.0-DRAFT" requirementId="CSIP5"/>
-  <!-- URL references to requirements for convenient lookup -->
+<testCase xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="testCase.xsd" testable="TRUE">
+  <id specification="CSIP" version="2.0.3" requirementId="CSIP5"/>
   <references>
-    <reference requirementId="CSIP5" URL="http://earkcsip.dilcis.eu/#CSIP5"/>
+    <reference requirementId="CSIP5" URL="https://earkcsip.dilcis.eu/#CSIP5"/>
   </references>
-  <!-- The full text of the requirement. -->
-  <requirementText>When the @csip:CONTENTINFORMATIONTYPE uses the value "OTHER" the @csip:OTHERCONTENTINFORMATIONTYPE must describe the content.
-</requirementText>
-  <!-- Textual description of the test case, extra notes beyond the requirment text. -->
-  <description></description>
-  <!-- List of requirments that this test case depends on in addition to the
-  main requirememt, e.g. general requirments on the form of IDs -->
+  <requirementText>
+      <name>Other Content Information Type Specification</name>
+      <location>mets[@csip:CONTENTINFORMATIONTYPE='OTHER']/@csip:OTHERCONTENTINFORMATIONTYPE</location>
+    <description>
+      When the mets/@csip:CONTENTINFORMATIONTYPE has the value "OTHER" the mets/@csip:OTHERCONTENTINFORMATIONTYPE must state the content information type.
+    </description>
+    <cardinality>0..1</cardinality>
+    <!--  It is actually a MUST or more precisely, a "conditional MUST":
```

Comment:
Yes - I think we need to discuss this with A2. How to represent conditional MUSTs in all of our specifications?

Reply:
I've created a ticket about it: https://github.com/DILCISBoard/E-ARK-CSIP/issues/657.